src/pages/FundingJoin/CompletePage.jsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import BackHeader from '../../components/common/BackHeader';
+import BottomBackground from '../../components/common/BottomBackground';
+import Button from '../../components/common/ButtonComponent';
+import { ReactComponent as GiftIcon } from '../../assets/FundingJoin/gift_icon.svg';
+
+const CompletePage = () => {
+  const Btn = <Button btnInfo={{ text: '홈으로 돌아가기', color: 'orange' }} />;
+
+  return (
+    <>
+      <BackHeader />
+      <SLayout>
+        <GiftIcon />
+        <STextContainer>
+          <SBigTextWrapper>펀딩 참여가 완료되었어요!</SBigTextWrapper>
+          <SSmallTextWrapper>
+            내가 참여한 펀딩은
+            <br />
+            [마이]-[내가 참여한 펀딩]에서
+            <br />
+            확인할 수 있어요
```

Comment:
`<br>` 태그는 요새 거의 안 쓰는 추세라고 알고 있어서, 스타일에서 줄바꿈 처리하면 좋을 거 같아요!
```suggestion
            {'내가 참여한 펀딩은\n[마이]-[내가 참여한 펀딩]에서\n확인할 수 있어요'}
```